static/app/views/dashboards/manage/dashboardList.tsx

```diff
@@ -32,22 +33,35 @@ import GridPreview from './gridPreview';
 
 type Props = {
   api: Client;
+  columns: number;
   dashboards: DashboardListItem[] | undefined;
   location: Location;
   onDashboardsChange: () => void;
   organization: Organization;
-  pageLinks: string;
+  rows: number;
+  isLoading?: boolean;
 };
 
 function DashboardList({
   api,
   organization,
   location,
   dashboards,
-  pageLinks,
   onDashboardsChange,
+  rows,
+  columns,
+  isLoading,
 }: Props) {
-  const navigate = useNavigate();
+  const [currentDashboards, setCurrentDashboards] = useState<
+    DashboardListItem[] | undefined
+  >(dashboards);
```

Comment:
Could you add a comment to explain why this cache exists? Just a line or two is fine